.cargo/config.toml

```diff
@@ -2,12 +2,9 @@
 target = "riscv64gc-unknown-none-elf"
 
 [target.riscv64gc-unknown-none-elf]
-rustflags = ["-C", "link-arg=-Tlinker.ld"]
+rustflags = [ "-C", "code-model=medium"]
```

Comment:
Please explain what's the importance of this.

Reply:
This ensures compatible when cross building and linking this. For pure rust, it hasnt been strictly, required. But I have seen errors regarding this a few times in the past weeks.  It says something like "symbols dont fit".

This setting, It aligns the code in s specific way when producing the elf. If you link many things together, you need to make sure they are aligned properly.

Reply:
Here they link to sysv abi https://doc.rust-lang.org/rustc/codegen-options/index.html#code-model